benchmark/single-source/AsyncTree.swift

```diff
@@ -0,0 +1,77 @@
+//===--- AsyncTree.swift -------------------------------------------===//
```

Comment:
```suggestion
//===--- AsyncTree.swift -------------------------------------------------===//
```

minor formatting fix, align to 80 chars